doc/cephadm/services/osd.rst

```diff
@@ -198,6 +198,18 @@ There are a few ways to create new OSDs:
 
 .. warning:: When deploying new OSDs with ``cephadm``, ensure that the ``ceph-osd`` package is not already installed on the target host. If it is installed, conflicts may arise in the management and control of the OSD that may lead to errors or unexpected behavior.
 
+* The OSDs created by ``ceph orch daemon add`` does not get added to OSD service by default. To add an OSD to OSD service
```

Comment:
OSDs created via ``ceph orch daemon add`` are by default not added to the orchestrator's OSD service.   To add an OSD to the OSD service issue a command of the following form: